### ISSUE TITLE
`git tag` seems to break it

### ISSUE THREAD

**Issue by thegoatherder** [JanDeDobbeleer/oh-my-posh#1665]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

Loving my oh-my-posh shell... until I ran `git tag`
Everything seems to fall apart. The powerlines get dismantled and the prompt start position is 30% into the terminal width. All the other commands I run seem to work fine.

![image](https://user-images.githubusercontent.com/389668/151636556-7886cade-7259-403a-b80c-e189aea2a2ae.png)

### Theme

```json
{
  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
  "console_title": true,
  "console_title_style": "template",
  "console_title_template": "{{ .Shell }} in {{ .Folder }}",
  "final_space": false,
  "blocks": [
    {
      "type": "prompt",
      "alignment": "left",
      "newline": true,
      "segments": [
        {
          "background": "#c386f1",
          "foreground": "#ffffff",
          "leading_diamond": "",
          "properties": {
            "display_host": false,
            "postfix": " "
          },
          "style": "diamond",
          "trailing_diamond": "",
          "type": "session"
        },
        {
          "background": "#ff479c",
          "foreground": "#ffffff",
          "powerline_symbol": "",
          "properties": {
            "folder_separator_icon": "  ",
            "home_icon": "~",
            "prefix": "   ",
            "style": "folder"
          },
          "style": "powerline",
          "type": "path"
        },
        {
          "background": "#fffb38",
          "foreground": "#193549",
          "powerline_symbol": "",
          "properties": {
            "ahead_and_behind_color": "#f26d50",
            "ahead_color": "#89d1dc",
            "behind_color": "#f17c37",
            "display_stash_count": true,
            "display_upstream_icon": true,
            "local_changes_color": "#ff9248",
            "stash_count_icon": " ",
            "status_colors_enabled": true
          },
          "style": "powerline",
          "type": "git"
        },
        {
          "background": "#6CA35E",
          "foreground": "#ffffff",
          "powerline_symbol": "",
          "properties": {
            "display_version": true,
            "prefix": "  "
          },
          "style": "powerline",
          "type": "node"
        },
        {
          "background": "#8ED1F7",
          "foreground": "#111111",
          "powerline_symbol": "",
          "properties": {
            "display_version": true,
            "prefix": "  "
          },
          "style": "powerline",
          "type": "go"
        },
        {
          "background": "#4063D8",
          "foreground": "#111111",
          "powerline_symbol": "",
          "properties": {
            "display_version": true,
            "prefix": "  "
          },
          "style": "powerline",
          "type": "julia"
        },
        {
          "background": "#FFDE57",
          "foreground": "#111111",
          "powerline_symbol": "",
          "properties": {
            "display_mode": "files",
            "display_version": true,
            "display_virtual_env": false,
            "prefix": "  "
          },
          "style": "powerline",
          "type": "python"
        },
        {
          "background": "#AE1401",
          "foreground": "#ffffff",
          "powerline_symbol": "",
          "properties": {
            "display_mode": "files",
            "display_version": true,
            "prefix": "  "
          },
          "style": "powerline",
          "type": "ruby"
        },
        {
          "background": "#FEAC19",
          "foreground": "#ffffff",
          "powerline_symbol": "",
          "properties": {
            "display_mode": "files",
            "display_version": false,
            "prefix": " "
          },
          "style": "powerline",
          "type": "azfunc"
        },
        {
          "background_templates": [
            "{{if contains \"default\" .Profile}}#FFA400{{end}}",
            "{{if contains \"jan\" .Profile}}#f1184c{{end}}"
          ],
          "foreground": "#ffffff",
          "powerline_symbol": "",
          "properties": {
            "display_default": false,
            "prefix": "  "
          },
          "style": "powerline",
          "type": "aws"
        },
        {
          "background": "#ffff66",
          "foreground": "#111111",
          "powerline_symbol": "",
          "properties": {
            "root_icon": ""
          },
          "style": "powerline",
          "type": "root"
        },
        {
          "background": "#83769c",
          "foreground": "#ffffff",
          "leading_diamond": "",
          "properties": {
            "always_enabled": true,
            "postfix": "⠀",
            "prefix": "<transparent></> 羽"
          },
          "style": "plain",
          "trailing_diamond": "",
          "type": "executiontime"
        },
        {
          "background": "#2e9599",
          "foreground": "#ffffff",
          "leading_diamond": "",
          "properties": {
            "always_enabled": true,
            "color_background": true,
            "display_exit_code": false,
            "error_color": "#f1184c",
            "prefix": "<#83769c></> "
          },
          "style": "diamond",
          "trailing_diamond": "",
          "type": "exit"
        }
      ]
    },
    {
      "type": "prompt",
      "alignment": "right",
      "segments": [
        {
          "background": "#0077c2",
          "foreground": "#ffffff",
          "properties": {
            "postfix": " <transparent,#0077c2></>",
            "prefix": "<#0077c2,transparent></>  "
          },
          "style": "plain",
          "type": "shell"
        },
        {
          "background": "#1BD760",
          "foreground": "#111111",
          "invert_powerline": true,
          "powerline_symbol": "",
          "properties": {
            "paused_icon": " ",
            "playing_icon": " ",
            "prefix": "  "
          },
          "style": "powerline",
          "type": "ytm"
        },
        {
          "type": "spotify",
          "style": "powerline",
          "invert_powerline": true,
          "powerline_symbol": "",
          "foreground": "#ffffff",
          "background": "#1BD760",
          "properties": {
            "prefix": "\uF9C6 ",
            "playing_icon": "\uE602 ",
            "paused_icon": "\uF8E3 ",
            "stopped_icon": "\uF04D ",
            "track_separator" : " - "
          }
        },
        {
          "background": "#f36943",
          "foreground": "#ffffff",
          "invert_powerline": true,
          "powerline_symbol": "",
          "properties": {
            "battery_icon": " ",
            "charged_icon": " ",
            "discharging_icon": " ",
            "charging_icon": " ",
            "charged_color": "#4caf50",
            "charging_color": "#4caf50",
            "color_background": true,
            "discharging_color": "#ff5722",
            "postfix": " <transparent></>"
          },
          "leading_diamond": "",
          "trailing_diamond": "",
          "style": "diamond",
          "type": "battery"
        },
        {
          "background": "#2e9599",
          "foreground": "#111111",
          "invert_powerline": true,
          "leading_diamond": "",
          "style": "diamond",
          "trailing_diamond": "",
          "type": "time"
        }
      ]
    },
    {
        "type": "prompt",
        "newline": true,
        "alignment": "left",
        "segments": [
            {
                "foreground": "#007ACC",
                "properties": {
                    "prefix": "",
                    "text": "PS❯"
                },
                "style": "plain",
                "type": "text"
            }
        ]
    }
  ]
}

```

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```shell
Version: 7.5.1

Segments:

ConsoleTitle(true)   -   0 ms - pwsh in PSProfile
session(true)        -   0 ms -  adamt 
path(true)           -   0 ms -    PSProfile 
git(true)            - 326 ms -   develop ≡ ~2 
node(false)          -   1 ms -
go(false)            -   0 ms -
julia(false)         -   0 ms -
python(false)        -   1 ms -
ruby(false)          -   0 ms -
azfunc(false)        -   0 ms -
aws(false)           -   0 ms -
root(false)          -   0 ms -
executiontime(true)  -   0 ms -  羽9.001s⠀
exit(true)           -   0 ms -   
shell(true)          -   0 ms -   pwsh 
ytm(false)           -  34 ms -
spotify(true)        -  27 ms - 阮  Dave Grusin - Fratelli Chase 
battery(false)       -   7 ms -
time(true)           -   0 ms -  23:49:06 
text(true)           -   3 ms - PS❯

Run duration: 407.9847ms

Cache path: C:\Users\adamt\AppData\Local/oh-my-posh

Logs:

2022/01/28 23:49:05 getArgs duration: 0s, args:
2022/01/28 23:49:05 getArgs duration: 0s, args:
2022/01/28 23:49:05 getShellName duration: 0s, args:
2022/01/28 23:49:05 debug: getenv

2022/01/28 23:49:05 getenv duration: 0s, args: OMP_CACHE_DISABLED
2022/01/28 23:49:05 isRunningAsRoot duration: 0s, args:
2022/01/28 23:49:05 getShellName duration: 0s, args:
2022/01/28 23:49:05 lastErrorCode duration: 0s, args:
2022/01/28 23:49:05 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:05 pwd duration: 0s, args:
2022/01/28 23:49:05 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getCurrentUser duration: 0s, args:
2022/01/28 23:49:05 getHostName duration: 0s, args:
2022/01/28 23:49:05 getRuntimeGOOS duration: 0s, args:
2022/01/28 23:49:05 templateCache duration: 0s, args:
2022/01/28 23:49:05 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:05 pwd duration: 0s, args:
2022/01/28 23:49:05 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:05 debug: getenv

2022/01/28 23:49:05 getenv duration: 0s, args: SSH_CONNECTION
2022/01/28 23:49:05 debug: getenv

2022/01/28 23:49:05 getenv duration: 0s, args: SSH_CLIENT
2022/01/28 23:49:05 getCurrentUser duration: 0s, args:
2022/01/28 23:49:05 getRuntimeGOOS duration: 0s, args:
2022/01/28 23:49:05 debug: getenv

2022/01/28 23:49:05 getenv duration: 0s, args: POSH_SESSION_DEFAULT_USER
2022/01/28 23:49:05 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:05 pwd duration: 0s, args:
2022/01/28 23:49:05 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:05 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:05 pwd duration: 0s, args:
2022/01/28 23:49:05 getArgs duration: 0s, args:
2022/01/28 23:49:05 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:05 getRuntimeGOOS duration: 0s, args:
2022/01/28 23:49:05 getRuntimeGOOS duration: 0s, args: 
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 getRuntimeGOOS duration: 0s, args:
2022/01/28 23:49:05 stackCount duration: 0s, args:
2022/01/28 23:49:05 templateCache duration: 0s, args:
2022/01/28 23:49:05 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:05 pwd duration: 0s, args:
2022/01/28 23:49:05 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:05 getRuntimeGOOS duration: 0s, args:
2022/01/28 23:49:05 hasCommand duration: 29.4531ms, args: git.exe
2022/01/28 23:49:05 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:05 pwd duration: 0s, args:
2022/01/28 23:49:05 hasParentFilePath duration: 0s, args: .git
2022/01/28 23:49:05 getFileContent duration: 529.5µs, args: C:\Users\adamt\OneDrive\Documents\PSProfile\.git/HEAD
2022/01/28 23:49:05 debug: runCommand
# branch.oid 71e7fa58143fff5cd412dba71f67997207a088b2
# branch.head develop
# branch.upstream origin/develop
# branch.ab +0 -0
1 .M N... 100644 100644 100644 96f282697ad6002ac64aa8376f54d1573a28c79d 96f282697ad6002ac64aa8376f54d1573a28c79d profile.ps1
1 .M N... 100644 100644 100644 7a1c8c6b5061f689c7e7e87c931eef308393a3ba 7a1c8c6b5061f689c7e7e87c931eef308393a3ba setup.ps1
2022/01/28 23:49:05 runCommand duration: 292.5513ms, args: git.exe -C C:\Users\adamt\OneDrive\Documents\PSProfile\ --no-optional-locks -c core.quotepath=false -c color.status=false status -unormal --branch --porcelain=2
2022/01/28 23:49:05 hasFolder duration: 581.9µs, args: C:\Users\adamt\OneDrive\Documents\PSProfile\.git/rebase-merge
2022/01/28 23:49:05 hasFolder duration: 574.7µs, args: C:\Users\adamt\OneDrive\Documents\PSProfile\.git/rebase-apply
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 hasFilesInDir duration: 512.3µs, args: MERGE_MSG
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 hasFilesInDir duration: 0s, args: CHERRY_PICK_HEAD
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 hasFilesInDir duration: 652.1µs, args: REVERT_HEAD
2022/01/28 23:49:05 getPathSeperator duration: 0s, args:
2022/01/28 23:49:05 hasFilesInDir duration: 0s, args: sequencer/todo
2022/01/28 23:49:06 error: getFileContent
open C:\Users\adamt\OneDrive\Documents\PSProfile\.git/logs/refs/stash: The system cannot find the file specified.
2022/01/28 23:49:06 getFileContent duration: 512.4µs, args: C:\Users\adamt\OneDrive\Documents\PSProfile\.git/logs/refs/stash
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 558.5µs, args: *.js
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 614.3µs, args: *.ts
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 0s, args: package.json
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 0s, args: .nvmrc
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 0s, args: pnpm-workspace.yaml
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 0s, args: .pnpmfile.cjs
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 0s, args: .npmrc
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 597.4µs, args: *.go
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 0s, args: go.mod
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 572µs, args: *.jl
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 563.1µs, args: *.py
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 513.6µs, args: *.ipynb
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 554.4µs, args: pyproject.toml
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 0s, args: venv.bak
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 0s, args: venv
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 0s, args: .venv
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 50.6µs, args: *.rb
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 516.1µs, args: Rakefile
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 126.9µs, args: Gemfile
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 0s, args: host.json
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 517.7µs, args: local.settings.json
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 getPathSeperator duration: 0s, args:
2022/01/28 23:49:06 hasFiles duration: 118.4µs, args: function.json
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 debug: getenv

2022/01/28 23:49:06 getenv duration: 0s, args: AWS_VAULT
2022/01/28 23:49:06 debug: getenv

2022/01/28 23:49:06 getenv duration: 0s, args: AWS_PROFILE
2022/01/28 23:49:06 debug: getenv

2022/01/28 23:49:06 getenv duration: 0s, args: AWS_REGION
2022/01/28 23:49:06 debug: getenv

2022/01/28 23:49:06 getenv duration: 0s, args: AWS_DEFAULT_REGION
2022/01/28 23:49:06 debug: getenv

2022/01/28 23:49:06 getenv duration: 0s, args: AWS_CONFIG_FILE
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 error: getFileContent
open C:\Users\adamt/.aws/config: The system cannot find the path specified.
2022/01/28 23:49:06 getFileContent duration: 0s, args: C:\Users\adamt/.aws/config
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 isRunningAsRoot duration: 0s, args:
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 executionTime duration: 0s, args:
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 lastErrorCode duration: 0s, args:
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 getShellName duration: 0s, args:
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 error: HTTPRequest
Get "http://127.0.0.1:9863/query": context deadline exceeded
2022/01/28 23:49:06 HTTPRequest duration: 34.059ms, args: http://127.0.0.1:9863/query
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 getWindowTitle duration: 27.0155ms, args: spotify.exe ^(Spotify.*)|(.*\s-\s.*)$
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 getBatteryInfo duration: 7.2736ms, args:
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 debug: pwd
C:\Users\adamt\OneDrive\Documents\PSProfile
2022/01/28 23:49:06 pwd duration: 0s, args:
2022/01/28 23:49:06 debug: homeDir
C:\Users\adamt
2022/01/28 23:49:06 templateCache duration: 0s, args:
2022/01/28 23:49:06 debug: getenv
C:\Users\adamt\AppData\Local
2022/01/28 23:49:06 getenv duration: 0s, args: LOCALAPPDATA
2022/01/28 23:49:06 getCachePath duration: 0s, args:
```

**Comment by lnu** [comment]
I tried your config on my side(macos):

https://user-images.githubusercontent.com/1829553/151657797-ab7da81b-b749-424d-8e35-93ca0535a951.mov

I'll reboot on windows to try the same thing:

![tmp](https://user-images.githubusercontent.com/1829553/151658054-4bbbdfc1-176a-4043-870d-f49f6cef20d1.gif)

Could be something with the repo. Do you have the issue with all repo?

**Comment by JanDeDobbeleer** [comment]
@thegoatherder I would first update git (if that's possible) as the pager in previous versions could influence the ANSI state in your terminal.

**Comment by thegoatherder** [comment]
@JanDeDobbeleer I think you're right. On further investigation, I found the issue was only happening in VSCode terminal and was triggered by most git commands, not just `git tag`. 

I was running Git 2.29.1. I have updated today to 2.35.1. The issue seems to have gone away.

I'll reopen this ticket if I find any similar issue, but for now I think upgrading Git is the correct solution.

Many thanks to you and @lnu for the rapid response! It's a fantastic piece of kit and has really improved my QOL. I salute you!
